### ISSUE TITLE
Improved Commit Titles\Doc Change Logs

### ISSUE THREAD

**Issue by LanceEa** [OfficeDev/office-js-docs#729]
**Issue:** 

The documentation is wonderful and appreciated! I spend a lot of time combing through it when working on my add-in. I know that the team is constantly improving it and enhancing it as Office Clients evolve. But, what I find difficult is understanding from week-to-week what has been changed or added to the docs.

Currently, I just take time going through and clicking each commit message to determine if it is something I'm interested in checking out. This is definitely time consuming. The reason I have to do this is that the Title's of the commits\pull request are not friendly nor do they identify what is being changed.

**Proposal:** 

I was wondering if the team would be able to adopt a more friendly commit\pull-request naming scheme? 

A good example of this is the [Angular Repo](https://github.com/angular/angular/commits/master). I can quickly see commits that I may be interested in reviewing such as _docs(router)_, _fix(router)_ etc... On the flip-side I can see the ones I might not be as interested in because they are internal implementation details such as _refactor(core)_ (well the nerd in me might care 😃). 

My current project is focused solely on an excel add-in so I would want to be able to glance at the commits and see if it was for excel or a general commit that effects all clients. I would then ignore looking at things for word or outlook, etc...

Potential naming scheme: 
{client} ({change classification}) - { additional info }

**Examples:**
- Excel (typo) - Fixed paragraph explaining dialogs
- Excel (examples) - added examples for creating add-ins with Angular 2 (now just Angular 🍺)
- Excel (events) - updated documentation on existing events
- Office (requirement-sets) - Updated requirement set table
- Word (OOXML) - added new apis to documentation.

I would ignore the word example and probably the typo example and check out the others. As explained above I have to click on every single one and only once it has loaded can I determine if I was interested in it. 

**Alternative Proposal**

Alternatively, providing a change log would work as well but I would think that would be more work for the team. Someone would have to go through and sift through the commits to create this since the naming scheme doesn't exist.

Let me know what you think about this and can definitely help brain-storming more.

**Comment by Lauragra** [comment]
Hi Lance, Thanks for the feedback. We'll take this back to the team and see what we can do here to make it easier for you to identify relevant content updates. We do publish what's new information for the Excel APIs in the [Excel JavaScript API requirement sets](https://dev.office.com/reference/add-ins/requirement-sets/excel-api-requirement-sets) article. Would it be helpful to you to see a last modified date on our topics on the site? That feature is currently on our roadmap. Please let us know if you have additional suggestions! Thanks.

**Comment by LanceEa** [comment]
Thanks for getting back to me so quickly! 

I'm familiar with the link you shared and do check that out. I definitely think that it is a helpful resource when quickly comparing what functionality is in which requirement set.  I do think having the last modified date is always helpful in documentation pages.

**Comment by kbrandl** [comment]
@LanceEa thanks for the suggestion, and apologies for the lack of engagement on this in recent months. We're planning to implement some new processes soon, along the lines of what you've discussed. I'm closing this issue now, but if you have additional feedback about the docs, please feel free to open a new issue in the [OfficeDev/office-js-docs-pr](https://github.com/OfficeDev/office-js-docs-pr) repo. Thanks!

**Comment by LanceEa** [comment]
@kbrandl  - no worries and thinks for the heads up!
